EditorJS/BlockHandler.php

```diff
@@ -136,6 +136,14 @@ private function validate($rules, $blockData)
                 continue;
             }
 
+            /**
+             * Do not check element type if it is not required and null
+             */
+            if (isset($rule['required']) && $rule['required'] === false &&
```

Comment:
@n0str I think it is enough with condition isset($rule['allow_null']) && $rule['allow_null'] === true && $value === null;

Reply:
so that it could be used without `required`